README.md

```diff
@@ -28,21 +32,37 @@ Also check the [config.yaml documentation](https://github.com/haskell/ghcup-hs/b
 
 ## For contributors
 
-### Adding a new GHC version
+Most bindists are built downstream by the GHCup developers, e.g.:
+
+* https://github.com/stable-haskell/haskell-language-server
+* https://github.com/stable-haskell/cabal
+* https://github.com/stable-haskell/stack
+
+For GHC bindists there is no automation yet and it is done manually (e.g. for FreeBSD and Alpine i386).
+
+That makes contributions harder, because your pull requests to the metadata will be partial.
```

Comment:
```suggestion
For GHC bindists there is no automation yet and it is done manually (e.g. for FreeBSD and Alpine i386).
That makes contributions harder, because your pull requests to the metadata will be partial.
```
If "that" refers to "no automation", this should be one paragraph. Otherwise, it looks strange.